src/JWT/JwtDecoder.cs

```diff
@@ -121,6 +105,6 @@ public void Validate(string[] parts, byte[] key)
             var decodedSignature = Convert.ToBase64String(signatureData);
 
             _jwtValidator.Validate(payloadJson, decodedCrypto, decodedSignature);
-        }
-    }
+        }       

```

Comment:
Extra spaces before line breaks. Please click Format Document in VS, press Ctrl+K+D (or K+E depending on the hotkeys layout)